components/mpas-ocean/cime_config/testdefs/testmods_dirs/mpaso/conservation_check/shell_commands

```diff
@@ -0,0 +1,2 @@
+# include mpas-ocean outputs in testing
+sed -i 's#compclass="ocn" exclude_testing="true"#compclass="ocn" exclude_testing="false"#g' env_archive.xml
```

Comment:
@jonbob and @rljacob, let me know what you think if this approach.  Happy to hear of other suggestions for how to not exclude MPAS-Ocean files from this particular test.

Reply:
Since its an xml file, can xmlchange do that?   If not, sed is fine.

Reply:
I couldn't figure out how.  I think `xmlchange` can only be used to modify specific fields that have an `id` and `value`, and this is not one of those.

Reply:
I slacked Jim to ask if he knows...